### ISSUE TITLE
Question about the LR schedule

### ISSUE THREAD

**Issue by MingSun-Tse** [shirley-wu/cross-doc-misinfo-detection#1]
Hi @shirley-wu , thanks for releasing the code for this inspiring project! I have one question about the LR schedule implementation.  [Here](https://github.com/shirley-wu/cross-doc-misinfo-detection/blob/25f10e61ae679bb7a57864f1a9e5da270c311a67/train.py#L55), it seems the arguments of `num_warmup_steps` and `num_training_steps` are switched by mistake. Consequently, the LR  is always arising during the whole training instead of a small warmup period. Just to confirm if this is a bug or I understand it wrong. 

Also, should the `num_training_steps` be 
> num_training_steps = args.num_epochs * len(dataloader_train) / args.accum-step ?

since every `args.accum-step` steps, the LR scheduler runs once, i.e., one real step. 

Looking forward to the feedback! Thanks!

**Comment by shirley-wu** [comment]
Hi, thank you so much for pointing it out! These are indeed bugs and I have fixed them in the code. You are welcome to double-check if it is correct.
